src/styles/global.ts

```diff
@@ -48,18 +48,23 @@ export const breakpoints = {
   sm: '768px' as const,
   md: '1024px' as const,
   lg: '1440px' as const,
+  customBp: '1217px' as const,
```

Comment:
This seems unnatural. Either add a new key `custom` and add `lg: 1217px` to that one. Or add it as a non-cutom attribute. Is the breakpoint part of design or made to remedy som issue with the implementation?

Reply:
I will look at it.

The breakpoint is for making sure the break happens before the metadata container gets too close to the rest of the content.